README.md

```diff
@@ -20,146 +23,171 @@ A small tool built to find and fix common misconfigurations in Active Directory
 ![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/jakehildreth/Locksmith/powershell.yml?logo=github&label=PSScriptAnalyzer)
 ![PowerShell Gallery Downloads](https://img.shields.io/powershellgallery/dt/locksmith?logo=powershell&label=PowerShell%20Gallery%20Downloads&color=blue)
 <!-- locksmith-badges-end -->
+
 ## Contents
+
 1. [Installation](#Installation)
-2. [Run Locksmith](#RunLocksmith)
-   1. [Mode 0](#Mode0)
-   2. [Mode 1](#Mode1)
-   3. [Mode 2](#Mode2)
-   4. [Mode 3](#Mode3)
-   5. [Mode 4](#Mode4)
+2. [Run Locksmith](#Run Locksmith)
+   1. [Mode 0](#Mode 0)
```

Comment:
Anchors can't have spaces.

Reply:
Ah dang, that's right. That's what I get for trying to make "cross-platform" anchor hotlinking (HTML hyperlink and Markdown hyperlink).